specs/deleteFromCart.spec.js

```diff
@@ -0,0 +1,44 @@
+let homePage = require('../pageObject/homePage');
+let summaryPage = require('../pageObject/summaryPage');
+
+
+describe('Delete from Cart', () => {
+    it('Positive', async () => {
+        await homePage.get();
+        await homePage.addToCart1stItem();
```

Comment:
make unique method like `addToCartItem(itemIndex)` instead of 
`await homePage.addToCart1stItem();
await homePage.addToCart2ndItem();
await homePage.addToCart3rdItem();`

Reply:
or `addToCartItem(itemName)`